.github/workflows/bump-version.yml

```diff
@@ -50,7 +50,7 @@ jobs:
       with:
         token: ${{ secrets.GIX_CREATE_PR_PAT }}
         branch: "release/v${{ steps.bump_version.outputs.new_version }}"
-        title: "Release: v${{ steps.bump_version.outputs.new_version }}"
+        title: "release: v${{ steps.bump_version.outputs.new_version }}"
```

Comment:
you don't need `release(something):` right?

Reply:
I'm pretty sure its not necessary. If I think about it I wouldn't really know what to put in the brackets.

release(oisy)? because its a release for everything right?

Reply:
i would say `release(ci)` or `ci(release)`

Reply:
correction: i saw that in the previous releases we would do `chore(release)` or `chore(devops)`